tuneme/templates/comments/comments.html

```diff
@@ -18,13 +18,13 @@
 <div class="pagination">
     <span class="step-links">
         {% if page.has_previous %}
-            <a href="?p={{ page.previous_page_number }}">&larr;</a>
+            <a href="?p={{ page.previous_page_number }}">&lt; &ndash; &ndash; &ndash;</a>
         {% endif %}
```

Comment:
@TrevorOctober Can you please remove the space between dashes, it should be something like ` &lt;&ndash;&ndash;&ndash; `